packages/react-accordion/src/components/AccordionItem/useAccordionItem.ts

```diff
@@ -35,10 +34,16 @@ export const useAccordionItem = (
 
   return {
     ...props,
-    ref,
     open,
-    onHeaderClick: onAccordionHeaderClick,
     disabled,
+    onHeaderClick: onAccordionHeaderClick,
     ...(navigable ? tabsterAttributes : {}),
+    root: resolveShorthand(props, {
+      required: true,
+      defaultProps: {
+        ref: ref,
+        ...(navigable ? tabsterAttributes : {}),
+      },
+    }),
```

Comment:
```suggestion
    root: getNativeElementProps({
        ...(navigable ? tabsterAttributes : {}),
        ...props
        ref,
    }),
```

This should use `getNativeElementProps()`